board/templates/home.html

```diff
@@ -576,6 +577,8 @@ <h5> Details</h5>\
                   var model_eval_iters = model_details[6]
```

Comment:
where is this variable? `model_details `

Reply:
Just 5 lines above that https://github.com/mohit-chawla/recboard/blob/5ad1ba5f07bfcf58042ca038186cabbb5266e10f/board/templates/home.html#L571